_includes/tutorials/confluent-parallel-consumer-application/kafka/code/src/main/java/io/confluent/developer/PropertiesUtil.java

```diff
@@ -0,0 +1,22 @@
+package io.confluent.developer;
+
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+
+/**
+ * Utility class for loading Properties from a file.
+ */
+public class PropertiesUtil {
+
+  public static Properties loadProperties(String fileName) throws IOException {
+    final Properties props = new Properties();
+    final FileInputStream input = new FileInputStream(fileName);
+    props.load(input);
+    input.close();
+    return props;
```

Comment:
We could use a `try-with-resources` block here.  But I'd say handle that in a follow-up PR

Reply:
@davetroiano I resolved this by accident - it's still relevant

Reply:
done